src/hera_display_games/random_walk/__init__.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A simple simulation of a random walk"""
+import click
+from hera_display_games.core import mechanics
+import time
+import random
+
+
+@click.command()
+@click.option("-s", "--speed", type=float, help="Speed of sprite")
+@click.option("-n", "--nsprites", type=int, help="number of sprites")
```

Comment:
These should probably have defaults? Or at least gracefully error with the docstring if not provided (currently it errors with a `TypeError: '<=' not supported between instances of 'int' and 'NoneType'`

Reply:
Added defaults.